src/containers/Login/index.js

```diff
@@ -2,45 +2,42 @@ import LoginComponent from "../../components/pages/Login";
 import actions from '../../actions/Login';
 import { connect } from "react-redux";
 import { useState } from "react";
+import styles from './style.module.css';
 
 function LoginContainer({ user, signIn }) {
-  const [values, setValues] = useState({ email: '', password: '' });
+  const [values, setValues] = useState({ name: '',email: '', password: '' });
 
-  // TODO: try this approach first
-  // function handleTextChange(event) {
-  //   setUser({ email: event.target.value, password: event.target.value });
-  // }
-
-  // TODO: then try this approach
-  // function handleTextChange(property) {
-  //   return function (event) {
-  //       setValues({ ...values, [property]: event.target.value });
-  //   }
-  // }
 
   const handleTextChange = (property) => (e) => {
     setValues({ ...values, [property]: e.target.value });
   }
 
   function handleLogin() {
-    signIn({ email: values.email, password: values.password })
+    signIn({ name: values.name, email: values.email, password: values.password })
+    window.localStorage.setItem('user', JSON.stringify(values)); 
```

Comment:
we can create a util class to handle the localStorage interaction, it can be useful